canonical-web-and-design/ubuntu.com
# CUBE: Change EDX API enrollments endpoint

## Done

The endpoint we were using is not stable in production so we are using
the older version of it as it is currently more stable.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube/microcerts
- Verify the page is still working and you are enrolled in the courses you are supposed to be enrolled in.
